go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 replace (
 	// dragonberry
 	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
-	github.com/cosmos/cosmos-sdk => github.com/scrtlabs/cosmos-sdk v0.45.11-0.20230123163150-1e11d183162f
+	github.com/cosmos/cosmos-sdk => github.com/scrtlabs/cosmos-sdk v0.45.11-0.20230123194044-5c4d6b629d9a
```

Comment:
From which branch was this taken, Itzik has some critical changes in a side branch @Cashmaney

Reply:
Yep

Reply:
This was the latest (https://github.com/scrtlabs/cosmos-sdk/commits/5c4d6b629d9a) on https://github.com/scrtlabs/cosmos-sdk/pull/257 before your rollback fixes.